### PR TITLE
COMMERCE-8365 Include urlTitleMap if it exists.

### PR DESCRIPTION
**Pull request by brianikim**
@riccardo-alberti 

https://issues.liferay.com/browse/COMMERCE-8365

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java [hunk 1]
```diff
@@ -456,7 +466,7 @@ else if (cpDefinition != null) {
 			product.getExternalReferenceCode(), commerceCatalog.getGroupId(),
 			LanguageUtils.getLocalizedMap(nameMap),
 			LanguageUtils.getLocalizedMap(shortDescriptionMap),
-			LanguageUtils.getLocalizedMap(descriptionMap), null,
+			LanguageUtils.getLocalizedMap(descriptionMap), urlTitleMap,
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:469]: Why don't you use LanguageUtils.getLocalizedMap method here?

Comment (brianikim) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:469]: hey @andrea-ale-sbarra , if CPDefinition is null, the value would be:
`Map<Locale, String> urlTitleMap = LanguageUtils.getLocalizedMap(productUrlTitleMap);`

If CPDefinition is not null, the value would be:
`urlTitleMap = cpDefinition.getUrlTitleMap();`

Correct me if I am wrong. Isn't cpDefinition.getUrlTitleMap() already localized?

Comment (andrea-ale-sbarra) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/ProductResourceImpl.java:469]: Yes you're right, but I think the same is for the other maps for example short description. I think either you change also all the others or you use the same pattern for this. I'm fine with both solutions

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**